doc/netopeer2-server.8

```diff
@@ -8,7 +8,7 @@ netopeer2-server \- NETCONF server daemon build on libnetconf2 with sysrepo data
 .
 .SH SYNOPSIS
 .B netopeer2-server
-[\fB-dhV\fP] [\fB-p\fP \fIPATH\fP] [\fB-U\fP[\fIPATH\fP]] [\fB-m\fP \fIMODE\fP] [\fB-u\fP \fIUID\fP]
+[\fB-dFhV\fP] [\fB-p\fP \fIPATH\fP] [\fB-U\fP[\fIPATH\fP]] [\fB-m\fP \fIMODE\fP] [\fB-u\fP \fIUID\fP]
```

Comment:
Please update the last change on line 5 as well for it to be obvious that is is up-to-date.

Reply:
Heh, of course! :-)